pkg/terraform/terraform.go

```diff
@@ -115,6 +118,10 @@ func unpackAndInit(dir string, platform string) (err error) {
 		return errors.Wrap(err, "failed to unpack Terraform modules")
 	}
 
+	if err := setupEmbeddedPlugins(dir); err != nil {
+		return errors.Wrap(err, "failed to setup embedded terraform plugins")
```

Comment:
Nit: "terraform" -> "Terraform"

Reply:
done with https://github.com/openshift/installer/pull/919/commits/80ef2116168dce89eb0c671fdb93a832a03a528c

Reply:
> Also, this is our only plugin setup, so we can probably drop embedded from both the function name and the error context.

but this informs that we are setting up the plugins that are embedded in out binary. IMO we shoud keep that.